src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -347,11 +349,19 @@ def translate_outputs(isce_output_dir: Path, product_name: str):
             file,
             dstSRS=f'epsg:{epsg}',
             creationOptions=['TILED=YES', 'COMPRESS=LZW', 'NUM_THREADS=ALL_CPUS'],
+            xRes=res,
+            yRes=res,
+            targetAlignedPixels=True
```

Comment:
@forrestfwilliams @hjkristenson any opinions on which [resampling method](https://gdal.org/programs/gdalwarp.html#cmdoption-gdalwarp-r) to use when reprojecting these geotiffs to their final UTM projection and pixel size?